src/containers/search-bar/store/sagas.js

```diff
@@ -22,10 +22,18 @@ export function* getSearchResults(searchText) {
   })
 
   const { tracks, albums, playlists, users } = results
+  const checkedUsers = users.filter(t => !t.is_deactivated)
   const checkedTracks = (yield call(setTracksIsBlocked, tracks)).filter(
-    t => !t.is_delete && !t._blocked
+    t => !t.is_delete && !t._blocked && !t.user.is_deactivated
   )
-  return { users, tracks: checkedTracks, albums, playlists }
+  const checkedPlaylists = playlists.filter(t => !t.user?.is_deactivated)
```

Comment:
since there's necessity to support this pattern for tracks, i kind of find this change to be super reasonable -- but maybe we should just have a generic helper that lives somewhere near-ish the cache that's like

checkUsers = (users: User[]) => users.filter(u => !u.is_deactivated)

and we just call that across the board (?)